src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -69,13 +70,19 @@ If enabled, the notification is suspended.
 
 If disabled or not configured, the notification is shown.
 
-Note: The notification about new major versions is always displayed.
+Notes: The notification about new major versions is always displayed. This policy has no effect if the update notifications are disable.
 </string>
       <string id="DisablePeriodicUpdateCheckDescription">This policy allows you to disable automatic update checks running in the background. (The manual check in PT Settings is not affected by this policy.)
 
 If enabled, the automatic update checks are disabled.
 
 If disabled or not configured, the automatic update checks are enabled.
+</string>
+      <string id="DisableNewUpdateToastDescription">This policy configures whether the action center notification for new updates is shown or not.
+
+If enabled, the notification is disabled.
+
+If disabled or not configured, the has the control whether the notification is shown or not.
```

Comment:
```suggestion
If enabled, the notification is hidden.

If disabled or not configured, the user has the control whether the notification is shown or not.
```

Reply:
It is not hidden. Ot is disabled and not shown.